include/envoy/http/header_map.h

```diff
@@ -236,6 +236,7 @@ class HeaderEntry {
   HEADER_FUNC(AccessControlExposeHeaders)                                                          \
   HEADER_FUNC(AccessControlMaxAge)                                                                 \
   HEADER_FUNC(Authorization)                                                                       \
+  HEADER_FUNC(B3)                                                                                  \
```

Comment:
This is not codified anywhere yet, but I would like to avoid adding extension headers to the core header map. We need a way to add O(1) headers to the map from extensions probably by static registration. This has come up before. In the interim I think a normal header search is probably OK so we can remove from here. Optimally, can we also move the other X-B3 headers at the same time?

Reply:
OK, ` We need a way to add O(1) headers to the map from extensions probably by static registration This has come up before`  Have related issue?I watch

Reply:
The need to complete in the separate PR?

Reply:
https://github.com/envoyproxy/envoy/issues/4815

Reply:
fixed